AniHyou/Screens/MediaList/Views/MediaListView.swift

```diff
@@ -52,15 +54,30 @@ struct MediaListView: View {
                         .tint(.blue)
                     }
                 }
-            }//:ForEach
-            
+
+            }
+                
             if viewModel.hasNextPage {
                 ProgressView()
                     .onAppear {
                         viewModel.getUserMediaList()
                     }
             }
         }//:List
+        .searchable(text: $searchText)
+        .refreshable {
+            viewModel.refreshList()
+        }
+        .onChange(of: viewModel.sort) { _ in
+            viewModel.refreshList()
+        }
+//        .onChange(of: viewModel.updatedEntry) { _ in
+//            viewModel.refreshList()
+//        }
```

Comment:
failed builds when not commented out... `updatedEntry` doesn't exist as a member of `viewModel`... guessing WIP?

Reply:
thi was a ugly workaround I deleted some weeks ago i this [commit](https://github.com/axiel7/AniHyou/commit/5810a84c990230191e06e485b74b52d2fda85541#diff-b2da5cd9e68800787127b85420f2cf2b351fb4b200c9c24d2e43af8be3eeb477). Maybe you didn't pull it?

Reply:
oh it might've been a merge conflict and I pulled it in by accident